src/sql/workbench/browser/designer/designerIssuesTabPanelView.ts

```diff
@@ -114,6 +126,13 @@ class TableFilterListRenderer implements IListRenderer<DesignerIssue, DesignerIs
 				break;
 		}
 		templateData.issueIcon.className = `issue-icon ${iconClass}`;
+		if (element.moreInfoLink) {
+			const linkElement = this._instantiationService.createInstance(Link, {
+				label: localize('designer.moreInfoLink', 'More information'),
```

Comment:
do you need an aria property?  or does that come from the label automatically?

Reply:
second parameter should be quoted using `""`

Reply:
@kburtram Links have aria-disabled by default: https://github.com/microsoft/azuredatastudio/blob/main/src/vs/platform/opener/browser/link.ts#L38